src/server/serverEntry.tsx

```diff
@@ -114,7 +114,7 @@ router.post(
 )
 
 router.get(
```

Comment:
hmm not sure what you mean, the "old URL" is now the landing page

Reply:
When we deploy it, would existing links from Hope or hedvig-com break?

Reply:
I can't see how anything would break 🤔 

From HOPE users will just land on this page instead of being redirected automatically.

It doesn't implicate hedvig-com